Adyen/HttpClient/Interfaces/IClient.cs

```diff
@@ -15,24 +15,43 @@
 //  *
 //  * Adyen Dotnet API Library
 //  *
-//  * Copyright (c) 2020 Adyen B.V.
+//  * Copyright (c) 2022 Adyen N.V.
 //  * This file is open source and available under the MIT license.
 //  * See the LICENSE file for more info.
 //  */
 #endregion
 
+using System;
 using System.Collections.Generic;
+using System.Net.Http;
 using System.Threading.Tasks;
 using Adyen.Model;
 
 namespace Adyen.HttpClient.Interfaces
 {
-    public interface IClient
+    public interface IClient : IDisposable
     {
-        string Request(string endpoint, string json, Config config);
-        string Request(string endpoint, string json, Config config, bool isApiKeyRequired, RequestOptions requestOptions);
+        /// <summary>
+        /// Request Method synchronous.
```

Comment:
Suggestion for summary: `Sends synchronous request to the specified <param refname="endpoint"/>.`